scripts/update-nightly/updateSubmodules.mjs

```diff
@@ -35,6 +35,7 @@ export async function updateSubmodules(newDefaultVersions, newNightlyVersions) {
     'docs/nightly/fuel-specs',
     'docs/nightly/fuel-graphql-docs',
     'docs/nightly/migrations-and-disclosures',
+    'docs/nightly/fuels-ts',
```

Comment:
This addition ensures that every call to the `update-nightly` workflow updates the nightly `fuels-ts` version. I only added `fuels-ts` and not the other repositories that have nightly versions because:
1. I want to keep the focus on the API docs issue which is `fuels-ts`-specific,
2. `fuels-ts` is the only repository in the `versions.json` file that calls the `update-nightly` workflow ([reference](https://github.com/search?q=org%3AFuelLabs+update-nightly&ref=advsearch&type=code)).

Handling of the other nightly versions more appropriately would be done via
- #570